cron/k8s/worker.release.yaml

```diff
@@ -29,6 +29,7 @@ spec:
       containers:
       - name: worker
         image: gcr.io/openssf/scorecard-batch-worker-releasetest:latest
```

Comment:
can we share the worker image for release test as well?

Reply:
Ah forgot. Done.